### DISCUSSION TITLE
Downloading from Amazon gives green video

### DISCUSSION THREAD

**Discussion by TheRicktatorship** [aclap-dev/video-downloadhelper#280] [Help and Q&A]
### Your question

I have a license and the companion app.  The download happens from Amazon with no issue but when I go to play it, the screen is green and there is no audio.

<img width="1280" alt="Screen Shot 2024-03-10 at 10 53 42 AM" src="https://github.com/aclap-dev/video-downloadhelper/assets/25229578/09e98a93-a7a3-47cd-81be-443c4fe157d5">

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 74bc1b2dc0ae4c8c
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/nickthomas",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "n6.1-11-gc30f360955"
}

**Comment by Mattmjs** [comment]
Nothing can be done about it, DRM protection.

**Comment by LJD1800** [comment]
I've had success with an app called Audials
